spec/factories/metrics.rb

```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :metric do
+    entity_key { 'generis' }
```

Comment:
I usually have fake data in my factories so my tests don't assume values from the factories (so they don't get coupled). What do you think?

Reply:
Yes, a good option is `Faker`. We should add it in all the factories, but that could be in another PR.

Reply:
@santib I agree that if a test asserts on a particular value it should be override the value in the same test instead of relying on the factory value.

@hdamico  Should I create a new ticket in the board for that?

Reply:
That would be appreciated!

Reply:
Added the task https://github.com/rootstrap/rs-code-review-metrics/issues/56 to the backlog.

Would you please review it when you have some time?

Thanks!